docs/bib/talks.bib

```diff
@@ -1,5 +1,16 @@
 % NB: entries with same author-title-year are not picked up:
 %     https://github.com/mcmtroffaes/sphinxcontrib-bibtex/issues/117
+@unpublished{Heinrich20181205,
+  title  = {Gaussian Process Smoothing for shape systematics estimation},
+  author = {Lukas Heinrich},
+  year   = {2018},
+  month  = {Dec},
+  day    = {5},
+  note   = {Joint Machine Learning & Statistics Fora Meeting},
```

Comment:
Can we also change this to match the other instance? That being

```
note   = {(Internal) Joint Machine Learning \& Statistics Fora Meeting},
```

This would then be consistent and then also not (I think? or maybe this gets skipped?) raise a warning in BibTex.